src/utils/useRowFinder.js

```diff
@@ -11,7 +11,14 @@ export default function useRowFinder() {
   // when the nav changes size, run this callback
   function callback([entry]) {
     // if there is nothing, skip it
-    if (!entry || !entry.target || !entry.target.children) return;
+    if (
+      !entry ||
+      !entry.target ||
+      !entry.target.children ||
+      !entry.borderBoxSize
+    ) {
```

Comment:
I added curly brackets here, because the if-condition spread over multiple lines. I find that easier to read then.